src/hooks/useSetFocus/__stories__/useSetFocus.stories.mdx

```diff
@@ -51,9 +51,9 @@ Hook for controlling focus on specific component e.g. Input.
       description={
         <>
           A React{" "}
-          <a href="https://react.dev/reference/react/useRef" target="_blank" rel="noopener noreferrer">
+          <Link to="https://react.dev/reference/react/useRef">
```

Comment:
```suggestion
          <Link href="https://react.dev/reference/react/useRef" withoutSpacing>
```

Reply:
Please also check if the text size of the link text and other text is the same. If it's bigger than might be needed to add `size={Link.sizes.SMALL}`

Reply:
Why you added without spacing in Hyperlink should that be added to all the hrefs..... that are replaced by <Link> tag